### DISCUSSION TITLE
You've used 100% of included services for the rmohamedshah account

### DISCUSSION THREAD

**Discussion by rmohamedshah** [community/community#33157] [Actions]
Hi,
 I am getting the below messages in the mail. Could you please let me know what is the step to clear the space.
 
 You've used 100% of included services for the rmohamedshah account 
 You've used 100% of included services for GitHub Actions
To continue using Actions & Packages uninterrupted, update your spending limit.

Your usage will reset on October 01, 2022.

Regards
Mohamed Shah R

**Comment by jfullstackdev** [comment]
maybe this thread will help 

https://github.com/orgs/community/discussions/26542

**Comment by airtower-luna** [comment]
The message you quoted sounds like it's not about disk space, but about runner minutes. For private repositories you only get a certain amount for free per month. If you need more, you need to either wait to the next month or pay for more. See: https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions

You can check what you have used in your account settings under "Billing and plans".
